docs/guide/cli/creating a Function.md

```diff
@@ -30,6 +30,12 @@ Once you are logged in, you can create a function by following these steps:
 -   Enter one of the available languages `go` `py` `js` `cs` `rb` `rs`
 -   Enter description `optional` `ascii` `max=80`
 
+::::tip tip
+
+When choosing a language make sure to use lowercase (this is different than when creating a function via calls to the API).
+
+::::
+
```

Comment:
it doesn't make much sense to add this, it is better to make the CLI handle this

Reply:
True, but until an update to the CLI is released to fix this issue its best to let people know?

Reply:
Then we can just remove this tip from the docs?

Reply:
The new version is coming out today or Monday so there's not much sense in having it

Reply:
Ah ok, will update. Thanks